### PR TITLE
Portfolio popup modal

### PR DESCRIPTION
**Pull request by HSMathebula**
##Portfolio

In this project we:

- Add a portfolio modal feature
- Add javaScript for the feature to popup

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -322,7 +323,78 @@ <h2 class="tittle t3">Get started</h2>
...
+                        <img class="demo cursor" src="./images/slipeimg.png" onclick="currentSlide(4)" alt="img">
+                      </div>
+                    </div>
+                </div>
+
+                <div class="bottom-div">
+                    <p class="p-tag">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, 
+                    </p>
+                    <p class="p-tag">
```
Comment (omar-muhamad) [inline index.html]: - [ ] Please remove the popup elements from your HTML file and kindly it suppose to be generated dynamically with javascript so please fix that.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by smunozmo — APPROVED** [review]
Hi @HSMathebula, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by khitermedachraf — COMMENTED** [review]
**Hello @HSMathebula 👋 you did Amazing work 🥇 .**
Clean Code 👍 .
Github workflow 👍 .
Here are some best practices maybe you can implement to improve your code:

1. it is best practice to begin committing messages with caps so kindly consider that in subsequent commits

1. Also, it is a best practice to use the kebab-case method when creat your name classes. and 

- [ ] keep going, Bro 🔥.

**Pull request merged**